README.md

```diff
@@ -1,11 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
Kindly fix conflicts in your PR. 
- Try fixing issues between the README.md that you have in your main branch and your deploy branch.
<img width="1440" alt="Screen Shot 2021-08-30 at 5 37 02" src="https://user-images.githubusercontent.com/22258274/131327020-474d223d-b614-4d71-a28c-b4aedf2dd737.png">